DOCS.md

```diff
@@ -1,27 +1,26 @@
-Use this plugin to upload files and build artifacts
-to the [Google Cloud Storage (GCS)](https://cloud.google.com/storage/) bucket.
+# GCS plugin
 
-You will need to a [Service Account](https://developers.google.com/console/help/new/#serviceaccounts)
-to authenticate to the GCS.
+Use this plugin to upload files and build artifacts to the [Google Cloud Storage (GCS)](https://cloud.google.com/storage/) bucket.
+
+You will need to a [Service Account](https://developers.google.com/console/help/new/#serviceaccounts) to authenticate to the GCS.
 
 The following parameters are used to configure this plugin:
 
-* `auth_key` - service account auth key
-* `source` - location of files to upload
-* `target` - destination to copy files to, including bucket name
-* `ignore` - skip files matching this [pattern](https://golang.org/pkg/path/filepath/#Match), relative to `source`
-* `acl` - a list of access rules applied to the uploaded files, in a form of `entity:role`
-* `gzip` - files with the specified extensions will be gzipped and uploaded with "gzip" Content-Encoding header
-* `cache_control` - Cache-Control header
-* `metadata` - an arbitrary dictionary with custom metadata applied to all objects
+* `source` - (Required) location of files to upload
+* `target` - (Required) destination to copy files to, including bucket name
+* `ignore` - (Optional) skip files matching this [pattern](https://golang.org/pkg/path/filepath/#Match), relative to `source`
+* `acl` - (Optional) a list of access rules applied to the uploaded files, in a form of `entity:role`
+* `gzip` - (Optional) files with the specified extensions will be gzipped and uploaded with "gzip" Content-Encoding header
+* `cache_control` - (Optional) Cache-Control header
+* `metadata` - (Optional) an arbitrary dictionary with custom metadata applied to all objects
+* `google_credentials` - (Required) is the GCP service account auth key
```

Comment:
This can be `auth_key` as well.

Reply:
Thanks. Missed that one. 

Maybe in the future it could be one of them?

Reply:
What do you mean it could be?

Reply:
Gotcha.
The best time to make that decision is now.
My preference is that all GCP plugins align on `GOOGLE_CREDENTIALS`.